lint-staged.config.js

```diff
@@ -3,12 +3,17 @@ module.exports = {
   '**/*.(ts|tsx)': () => 'yarn tsc --noEmit',
 
   // Lint then format TypeScript and JavaScript files
-  '**/*.(ts|tsx|js)': (filenames) => [
+  '**/*.(ts|tsx|js|jsx)': (filenames) => [
     `yarn eslint --fix ${filenames.join(' ')}`,
     `yarn prettier --write ${filenames.join(' ')}`,
   ],
 
+  // Run unit tests relating to modified files
+  '**/*.(ts|tsx|js|jsx)': (filenames) => [
+    `jest --findRelatedTests ${filenames.join(' ')}`,
+  ],
+
```

Comment:
Run unit tests related to staged files?  Helpful?  Or annoying?

Reply:
we'll soon find out :D